RecoBTag/DeepFlavour/plugins/BTagConverter.h

```diff
@@ -0,0 +1,37 @@
+#ifndef RecoBTag_DeepFlavour_BTagConverter_h
+#define RecoBTag_DeepFlavour_BTagConverter_h
+
+#include "RecoBTag/DeepFlavour/plugins/deep_helpers.h"
```

Comment:
[minor] this can now be simply ```#include "deep_helpers.h"```

Reply:
ok, I will simplify it (and the other converters) when I update to the new PtCut model